src/main/java/com/minecolonies/coremod/client/gui/WindowRequestDetail.java

```diff
@@ -206,6 +230,49 @@ public void onOpened()
         box.setSize(box.getWidth(), y);
     }
 
+    /**
+     * Checks if the request is fulfillable
+     */
+    public Boolean fulfillable()
```

Comment:
I have a feeling that some checks are missing here compared to the checks we got in the abstract class.
I'd probably move the checks from the abstract class also into a separate method to allow to call it from here.